subjects/java/checkpoints/double-linkedlist/README.md

```diff
@@ -0,0 +1,144 @@
+## Double Linked List
+
+### Instructions
+
+Create a doubly linked list data structure that implements the following methods:
+
+- `at(int index)`: to access an element by its index.
+- `add(int value)`: to add an element at the end of the list.
+- `remove(int index)`: to remove an element by its index.
+- `size()`: to get the size of the list.
+
+Define these methods in an interface called `LinkedList`, and implement this interface in a class that you will create. Additionally, add private methods `next(Node node)` and `prev(Node node)` in your class that will be used in the other methods. These methods should print the messages "Go to next node" and "Go to previous node" each time they are called, respectively.
```

Comment:
When does `prev` is supposed to be called instead of `next`? I think this info should be clear since it will be tested how many time a method is called

Reply:
yeah they have to use it to have fast access to the elements. 
i guess it's clear from the test examples.